rental.rb

```diff
@@ -0,0 +1,12 @@
+class Rental
+  attr_accessor :date
```

Comment:
- [ ] @book and @person suppose to have a getter and setter method. Kindly implement this by using the attribute accessor. 👍 
```suggestion
class Rental
  attr_accessor :date, :book, :person
```

Reply:
Thanks so much for the clarity @OyePriscilla